crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -1112,10 +1112,25 @@ impl SuiTransactionBlock {
 
 impl Display for SuiTransactionBlock {
     fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
-        let mut writer = String::new();
-        writeln!(writer, "Transaction Signature: {:?}", self.tx_signatures)?;
-        write!(writer, "{}", &self.data)?;
-        write!(f, "{}", writer)
+        let mut builder = TableBuilder::default();
+
+        builder.push_record(vec![format!("{}", self.data)]);
+        builder.push_record(vec![format!("Signatures:")]);
+        for tx_sig in &self.tx_signatures {
+            builder.push_record(vec![format!(
+                "   {}\n",
+                // this does not return the same sig string as the explorer?!?!
+                Base64::from_bytes(tx_sig.as_ref()).encoded()
```

Comment:
Is this the correct way to get the Base64 of a `GenericSignature`? @joyqvq @mskd12 
Probably here we want to have a Display trait on GenericSignature...